scarb/src/ops/package.rs

```diff
@@ -147,15 +198,16 @@ fn package_one_impl(
 
 fn list_one_impl(
     pkg_id: PackageId,
-    _opts: &PackageOpts,
+    opts: &PackageOpts,
     ws: &Workspace<'_>,
 ) -> Result<Vec<Utf8PathBuf>> {
     let pkg = ws.fetch_package(&pkg_id)?;
-    let recipe = prepare_archive_recipe(pkg)?;
+    let vcs_info = extract_vcs_info(pkg, opts)?;
```

Comment:
you can put this line directly within `prepare_archive_recipe`; no need to put it outside